dev_utils/README.md

```diff
@@ -1,20 +1,33 @@
 # Dev environment setup recomendations
 
-This guide uses the
-[minio client](https://docs.min.io/minio/baremetal/reference/minio-cli/minio-mc.html)
-(mc) for testing.
+## Deploy a stack locally
 
-## minio s3 server
+To start the S3Proxy development environment locally with docker compose, run the following command from the directory `dev_utils`
+
+```bash
+docker compose run local
+```
+
+After that, you can use [s3cmd](https://s3tools.org/s3cmd) to manually interact with the s3 server with proxy by 
+
+```bash
+s3cmd -c proxyS3 put README.md s3://dummy ## Upload a file using the proxy
+s3cmd -c proxyS3 ls s3://dummy ## List all files of the user using the proxy 
+```
 
-The S3Proxy development environment is run through docker compose, and can be
-started from this directory using:
+>Note that the content of the file `proxyS3` will be modified since the the string `TOKEN` will be replaced by the actual token during the local deployment. Make sure not to commit this change.
```

Comment:
```suggestion
>Note that the content of the file `proxyS3` will be modified since the string `TOKEN` will be replaced by the actual token during the local deployment. Make sure not to commit this change.
```